sshconnect.c

```diff
@@ -500,8 +506,13 @@ ssh_connect_direct(struct ssh *ssh, const char *host, struct addrinfo *aitop,
 				break;
 			} else {
 				oerrno = errno;
-				debug("connect to address %s port %s: %s",
-				    ntop, strport, strerror(errno));
+				if (ai->ai_family != AF_UNIX) {
+					debug("connect to %s port %s: %s",
+					    ntop, strport, strerror(errno));
+				} else {
+					debug("connect to %s: %s",
+					    host, strerror(errno));
```

Comment:
Remove these three lines, as the same message will anyway be printed as an error immediatelly after. (for unix domain sockets, we always have exactly one entry in the struct addrinfo linked list)

Reply:
I'd prefer to leave it there, for symmetry reasons.